docs/paper/dev/api/command-api/arguments/entity-player-arguments.mdx

```diff
@@ -0,0 +1,175 @@
+<Javadoc name="io.papermc.paper.command.brigadier.argument.resolvers.ArgumentResolver#resolve(io.papermc.paper.command.brigadier.CommandSourceStack)">`ArgumentResolver#resolve(CommandSourceStack)`</Javadoc>,
+which returns a `List<Entity>`.
+
+### Example usage
+```java
+public static LiteralCommandNode<CommandSourceStack> entities() {
+    return Commands.literal("entitiesarg")
+        .requires(ctx -> ctx.getSender().isOp())
+        .then(Commands.argument("arg", ArgumentTypes.entities())
+            .executes(ctx -> {
+                final EntitySelectorArgumentResolver entitySelectorArgumentResolver = ctx.getArgument("arg", EntitySelectorArgumentResolver.class);
+                final List<Entity> entities = entitySelectorArgumentResolver.resolve(ctx.getSource());
+
+                ctx.getSource().getSender().sendRichMessage("Found the following entities: <entitytypes>",
+                    Placeholder.unparsed("entitytypes", String.join(", ", entities.stream().map(Entity::getType).map(EntityType::name).toList()))
+                );
+                return Command.SINGLE_SUCCESS;
+            }))
+        .build();
+}
+```
+
+### In-game preview
+<FullWidthVideo src={EntitiesMp4}/>
+
+
+## Player Argument
+The player argument allows to retrieve a `PlayerSelectorArgumentResolver` for player arguments.
+For this "single player" argument, you can safely get the target player by running `PlayerSelectorArgumentResolver.resolve(ctx.getSource()).getFirst()`,
+which returns a <Javadoc name='org.bukkit.entity.Player'>Player</Javadoc> object.
+
+### Example usage
+This command yeets the targeted player into the air!
+
+```java
+public static LiteralCommandNode<CommandSourceStack> playerArgument() {
+    return Commands.literal("player")
+        .then(Commands.argument("target", ArgumentTypes.player())
+            .executes(ctx -> {
+                final PlayerSelectorArgumentResolver targetResolver = ctx.getArgument("target", PlayerSelectorArgumentResolver.class);
+                final Player target = targetResolver.resolve(ctx.getSource()).getFirst();
+
+                target.setVelocity(new Vector(0, 100, 0));
+                target.sendRichMessage("<rainbow>Yeeeeeeeeeet");
+
+                ctx.getSource().getSender().sendRichMessage("Yeeted <target>!",
+                    Placeholder.component("target", target.name())
+                );
+                return Command.SINGLE_SUCCESS;
+            }))
+        .build();
+}
+```
+
+### In-game preview
+<FullWidthVideo src={PlayerMp4}/>
+
+
+## Players Argument
+The "multiple players" argument works similarly to the "single player" argument, also returning a `PlayerSelectorArgumentResolver`. Instead of just resolving to exactly one `Player`, this
+one can resolve to more than just one player - which you should account for in case of using this argument. `PlayerSelectorArgumentResolver.resolve#(ctx.getSource())` returns a

```

Comment:
```suggestion
one can resolve to more than just one player - which you should account for in case of using this argument. `PlayerSelectorArgumentResolver.resolve(ctx.getSource())` returns a
```